.github/workflows/release.yml

```diff
@@ -106,6 +106,8 @@ jobs:
       - name: 'Create Temporary Branch'
         run: |
           git checkout -b "docs-$GITHUB_SHA"
+      - name: Install GraphViz
+        run: sudo apt install --yes graphviz
```

Comment:
Don't you need to call `update` before, and isn't it `apt-get`?

Reply:
Well, "need" is too hard.
Update updates the package index first.
So without "update" you might install an older version than available and at some point in the future it might fail.
Currently, it worked without.
But I added it now as I indeed was not intentional. :-)

Regarding `apt` vs. `apt-get`, both would work.
Since roundabout 10 years there is `apt` which you usually use nowadays, at least as human.
It replaces `apt-get` and `apt-cache` as frontend utility, leaving those as more low-level backend commands that are more useful in scripts that parse the output.